tinker/admin/program_search/__init__.py

```diff
@@ -66,22 +66,28 @@ def submit(self):
 
     @route('/multi-delete', methods=['POST'])
     def multi_delete(self):
-        ids_to_delete = json.loads(request.data)
-        for id in ids_to_delete:
-            if not (isinstance(id, str) or isinstance(id, unicode)):
+        list_of_ids_to_delete = json.loads(request.data)
```

Comment:
Line 69, right side: This `json.loads` parses into a list object, so it can't be wrapped in an EncodingDict. As such, I encode any unicode to String inside the for loop below. This is the only place in Tinker that I was unable to use EncodingDict.